docs/Account/methods.md

```diff
@@ -0,0 +1,284 @@
+
+type UserOpReceipt = {
+  userOpHash: string;
+  entryPoint: string;
+  sender: string;
+  nonce: number;
+  paymaster: string;
+  actualGasCost: BigNumber;
+  actualGasUsed: BigNumber;
+  success: boolean;
+  reason: string;
+  logs: Array<ethers.providers.Log>;
+  receipt: ethers.providers.TransactionReceipt;
+};
+```
+
+**Parameters**
+- userOp (`Partial<UserOperation>`, required): The `userOp` object includes essential fields like `sender`, `nonce`, `callData`, `gas` related properties.
+
+- params (`SendUserOpParams`): The `SendUserOpParams` object can contain fields such as `sessionID`, `sessionSigner`, `sessionValidationModule`, `additionalSessionData`, `batchSessionParams`, and `simulationType`. These parameters are used to customize the behavior of the `sendUserOp` method and are optional. 
+  ```ts
+  const userOpResponse = await moduleSmartAccount?.sendUserOp(userOp, {
+    sessionSigner: sessionSigner,
+    sessionValidationModule: moduleAddr,
+  });
+  ```
+  Similar to building a `userOp` we need to ensure that any modules used for additional validation or execution logic are specified in the `sendUserOp` method. Currently, this only applies for session key module requirements.
+
+:::note
+Please note that `simulationType` allows for more debugging insights about `callData` on why an internal transaction fails. It is set to "validation" by default, but can be changed to "validation_and_execution" for more detailed tracing.
+:::
+
+**Returns**
+- userOpsResponse (`UserOpResponse`): The method returns an object of type `UserOpResponse` which has a `userOpHash` and two methods: `wait()` and `waitForTxHash()`.
+The `wait()` method resolves when the user operation is dispatched by the bundler on-chain and gets mined. The `waitForTxHash()` method returns a `UserOpStatus` object which includes the transaction hash and the receipt once added on-chain.
+
+
+### sendSignedUserOp( )
+
+This method is designed to handle user operations that have already been signed. These signed operations are sent directly to the bundler for on-chain processing.
+This method is particularly useful when handling operations that have been grouped together with a multi-chain module, as it allows for the submission of these combined operations in a single request.
+
+**Usage**
+```ts
+const userOpResponse = await smartAccount.sendSignedUserOp(userOp)
+```
+:::note
+Please ensure that the user operations have been correctly signed before using this method⁠
+:::
+**Parameters**
+- userOp (`UserOperation`, required): The `userOp` object includes essential fields like `sender`, `nonce`, `callData`, `gas` related properties, and `signature`.
+
+- params (`SendUserOpParams`): The `SendUserOpParams` object can contain fields such as `sessionID`, `sessionSigner`, `sessionValidationModule`, `additionalSessionData`, `batchSessionParams`, and `simulationType`. These parameters are used to customize the behavior of the `sendUserOp` method and are optional. 
```

Comment:
I think only simulationType is relevant here because we won't be using the rest. since userOp is already signed we don't need to send it to module for signing hence additional params won't be needed. that will confuse the users

@VGabriel45 we can later replace the type with something else. currently it is shared with params type of sendUserOp only

Reply:
should I remove the params from here?

Reply:
@livingrockrises "we can later replace the type with something else. currently it is shared with params type of sendUserOp only" can you elaborate on this ? Maybe on Slack if you don't want to extend the chat here.

Reply:
explained on slack